frontend/dashboard/components/ResourcesRepoList/ResourcesRepoList.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { RESOURCEADM_BASENAME } from 'app-shared/constants';
+import { useSelectedContext } from 'dashboard/hooks/useSelectedContext';
+import { ResourceTable } from 'resourceadm/components/ResourceTable';
+import { useGetResourceListQuery } from 'resourceadm/hooks/queries/useGetResourceListQuery';
+import { getResourceDashboardURL, getResourcePageURL } from 'resourceadm/utils/urlUtils';
+import { getReposLabel } from 'dashboard/utils/repoUtils';
+import { Organization } from 'app-shared/types/Organization';
+import { useTranslation } from 'react-i18next';
+import { SelectedContextType } from 'app-shared/navigation/main-header/Header';
+import { AltinnSpinner } from 'app-shared/components';
+
+type ResourcesRepoListProps = {
+  organizations: Organization[];
+};
+
+export const ResourcesRepoList = ({ organizations }: ResourcesRepoListProps): React.ReactNode => {
+  const { t } = useTranslation();
+  const selectedContext = useSelectedContext();
+  const repo = `${selectedContext}-resources`;
+
+  const isOrganisation =
+    selectedContext !== SelectedContextType.All && selectedContext !== SelectedContextType.Self;
+  const { data: resourceListData, isLoading } = useGetResourceListQuery(
+    selectedContext,
+    !isOrganisation,
+  );
+
+  if (!isOrganisation) {
+    return null;
+  }
+
+  return (
+    <div data-testid='resource-table-wrapper'>
+      <h2>
+        {getReposLabel({
+          selectedContext,
+          orgs: organizations,
+          t,
+          isResourcesRepo: true,
+        })}
+      </h2>
+      {isLoading ? (
+        <AltinnSpinner spinnerText={t('general.loading')} />
+      ) : (
+        <div>
+          <a href={`${RESOURCEADM_BASENAME}${getResourceDashboardURL(selectedContext, repo)}`}>
+            {t('dashboard.go_to_resources')}
+          </a>
+          <ResourceTable
+            list={resourceListData || []}
```

Comment:
Why do we need to fallback to empty array here? If you do not have any resourceList, the backend should return empty array by default? 🤔

Reply:
Backend returns 204 on no resources, so we need to fallback to `[]` since resourceListData will be `null` in this case.

Reply:
@simen-rekkedal, I suggest that we change the endpoint to return 200 OK but with an empty array [].  What do you think? This will make the frontend a little simplified. 🤩

Reply:
@framitdavid  https://github.com/Altinn/altinn-studio/pull/11686

Reply:
@mgunnerud do you change the implementation within the frontend when the backend PR is merged? 🥳